src/web3/utils.js

```diff
@@ -16,3 +16,18 @@ export const validateBlockNumber = (lastBlockNumber, blockNumber) => {
     throw new Error(`${blockNumber} ${lastBlockNumber} is not valid block number`);
   }
 };
+
+/**
+ * Return the eth network id for the current web3Instance
+ * @returns integer
+ */
+const getEtherNetworkId = () => web3Instance.version.network;
+
+const web3Utils =
+{
+  isAddress,
+  validateBlockNumber,
+  getEtherNetworkId,
+};
+
+export default web3Utils;
```

Comment:
why not just export each function separately?

Reply:
By design 
sinon doesn't stub methods without them being wrapped in an Object thttps://github.com/sinonjs/sinon/issues/562

The other solutions provided are very dirty honestly. So, we either keep it like this or we make etherscan module take an extra networkID param and pass through to this method.